File: apps/beavernest-be-e2e/steps/development.steps.ts
/**
 * Aggregate binding for the development-wrapper contract. The executable
 * shell contract owns the filesystem assertions; browser E2E keeps the shared
 * feature represented without introducing a production test route.
 */
import { createBdd } from "playwright-bdd";

const { Given, When, Then } = createBdd();

Given("the local development command receives an explicit developer-owned data directory", async () => undefined);
When("it starts the backend on the local development port", async () => undefined);
Then("the database resolves only within that development directory", async () => undefined);
Then("the command neither reads nor inherits the production host data-bind source", async () => undefined);

File: apps/beavernest-be/tests/unit/Steps/DevelopmentSteps.fs
module BeaverNestBe.Tests.Unit.Steps.DevelopmentSteps

open TickSpec

[<Given>]
let ``the local development command receives an explicit developer-owned data directory`` () = ()

[<When>]
let ``it starts the backend on the local development port`` () = ()

[<Then>]
let ``the database resolves only within that development directory`` () = ()

[<Then>]
let ``the command neither reads nor inherits the production host data-bind source`` () = ()
